plugins/java-decompiler/engine/README.md

```diff
@@ -14,8 +14,8 @@ Fernflower is licenced under the Apache Licence Version 2.0. It can be found her
 
 java -jar fernflower.jar [-<option>=<value>]* [<source>]+ <destination>
 
-* means 0 or more times
-+ means 1 or more times
+\* means 0 or more times\
```

Comment:
No, you're right. Github makes this into two lines, but some other formatters merge them into one. There are other issues in the file - I'll clean 'em up after merging the PR. Thanks!